ospf6d/ospf6_top.c

```diff
@@ -911,6 +913,73 @@ DEFUN (no_ospf6_stub_router_admin,
 	return CMD_SUCCESS;
 }
 
+/* Restart OSPF SPF algorithm*/
+static void ospf6_restart_spf(struct ospf6 *ospf6)
+{
+	/* Handling inter area and intra area routes*/
+	ospf6_route_remove_all(ospf6->route_table);
+	ospf6_route_remove_all(ospf6->brouter_table);
+	ospf6_route_remove_all(ospf6->external_table);
+
+	/* Trigger SPF */
+	ospf6_spf_schedule(ospf6, OSPF6_SPF_FLAGS_CONFIG_CHANGE);
+}
+
+/* Set the max paths */
+static void ospf6_maxpath_set(struct ospf6 *ospf6, uint16_t paths)
+{
+	if (ospf6->max_multipath == paths)
+		return;
+
+	ospf6->max_multipath = paths;
```

Comment:
no, really: you _have_ to cap the max at MULTIPATH_NUM. you cannot allow a config value greater than that.

Reply:
I found a way to fix it and add the max values also at the compile time. I have done. Pls review.